src/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliEncoder.cs

```diff
@@ -92,19 +92,19 @@ internal void SetWindow(int window)
             }
         }
 
-        public static int GetMaxCompressedLength(int length)
+        public static int GetMaxCompressedLength(int inputSize)
         {
-            if (length < 0 || length > BrotliUtils.MaxInputSize)
+            if (inputSize < 0 || inputSize > BrotliUtils.MaxInputSize)
             {
-                throw new ArgumentOutOfRangeException(nameof(length));
+                throw new ArgumentOutOfRangeException(nameof(inputSize));
```

Comment:
I would have hoped that this would break a test that was validating the name stored in the exception.  If we have a test for GetMaxCompressedLength with an out-of-range input and it's just not validating the parameter name, can you augment it?  And if we don't, can you add one?  Thanks.

Reply:
Found a couple. Fixed. Thanks for letting me know.